helix-term/src/application.rs

```diff
@@ -83,13 +83,14 @@ impl Application {
                 editor.new_file(Action::VerticalSplit);
                 compositor.push(Box::new(ui::file_picker(first.clone())));
             } else {
+                editor.open(first.to_path_buf(), Action::VerticalSplit)?;
                 for file in args.files {
                     if file.is_dir() {
                         return Err(anyhow::anyhow!(
                             "expected a path to file, found a directory. (to open a directory pass it as first argument)"
                         ));
                     } else {
-                        editor.open(file, Action::VerticalSplit)?;
+                        editor.open(file, Action::Replace)?;
```

Comment:
Should we show the first file in the argument list or the last one? With this change it will continually call replace and so we'll end up on the last file. Maybe that's the right behavior though

Reply:
Showing the first file is the default behavior in most editors (vim, nano, vscode, etc) so doing that would be the least surprising.

Reply:
Fair enough, the order is now changed. The first file given on the commandline is now the file on display.